src/pages/fr/core-concepts/sharing-state.md

```diff
@@ -0,0 +1,698 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: État de partage
+i18nReady: true
+setup: |
+  import UIFrameworkTabs from '~/components/tabs/UIFrameworkTabs.astro'
+  import LoopingVideo from '~/components/LoopingVideo.astro'
+  import JavascriptFlavorTabs from '~/components/tabs/JavascriptFlavorTabs.astro'
+---
+
+Lors de la construction d'un site Web Astro [architecture Islands / hydratation partielle](/fr/concepts/islands/), Vous avez peut-être rencontré ce problème: **Je veux partager l'état entre mes composants.**
+
+Les frameworks d'interface utilisateur comme React ou Vue peuvent encourager ["context" providers](https://reactjs.org/docs/context.html). Mais quand les [composants partiellement hydratés](/fr/core-concepts/framework-components/#hydrating-interactive-components) dans Astro ou Markdown, vous ne pouvez pas utiliser ces emballages de contexte..
+
+Astro recommande une solution différente pour le stockage partagé côté client: [**Nano Stores**](https://github.com/nanostores/nanostores).
+
+## Pourquoi Nano Stores?
+
+La librairie [Nano Stores](https://github.com/nanostores/nanostores) vous permet de créer des magasins avec lesquels n'importe quel composant peut interagir. Nous recommandons Nano Stores car
+:
+- **Ils sont légers.** Nano Stores expédie le strict minimum JS dont vous aurez besoin (moins de 1 Ko) sans aucune dépendance.
+- **Ils sont indépendants du framework.** Cela signifie que le partage d'état entre les frameworks sera transparent ! Astro est basé sur la flexibilité, nous aimons donc les solutions qui offrent une expérience de développeur similaire, quelle que soit votre préférence.
+
+Pourtant, il existe un certain nombre d'alternatives que vous pouvez explorer. Ceux-ci inclus:
+- [Svelte's built-in stores](https://svelte.dev/tutorial/writable-stores)
+- [Solid signals](https://www.solidjs.com/docs/latest) en dehors d'un contexte de composant
+- [Envoi d'événements de navigateur personnalisés](https://developer.mozilla.org/fr/docs/Web/Events/Creating_and_triggering_events) entre les composants
```

Comment:
Nit: Don't you think "Envoi" needs an article here?

Reply:
```suggestion
- [L'envoi d'événements de navigateur personnalisés](https://developer.mozilla.org/fr/docs/Web/Events/Creating_and_triggering_events) entre les composants
```